language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -1998,4 +2000,94 @@ public static Optional<Node> getMatchingNode(Node currentNode, Predicate<Node> p
     public static boolean isWithInRange(Node node, int offset) {
         return node.textRange().startOffset() <= offset && offset <= node.textRange().endOffset();
     }
+
+    /**
+     * Get the list of function arguments from the invokable symbol.
+     *
+     * @param symbol Invokable symbol to extract the arguments
+     * @param ctx    Lang Server Operation context
+     * @return {@link List} List of arguments
+     */
+    public static List<String> getFuncArguments(FunctionSymbol symbol, BallerinaCompletionContext ctx) {
+        List<String> args = new ArrayList<>();
+        Pair<List<ParameterSymbol>, Optional<ParameterSymbol>> params = CommonUtil.getFunctionParameters(symbol, ctx);
+        List<ParameterSymbol> reqParams = params.getLeft();
+        for (ParameterSymbol param : reqParams) {
+            args.add(getParamTypeAndNamePair(param, ctx).orElse(""));
+        }
+        if (params.getRight().isPresent()) {
+            args.add(getParamTypeAndNamePair(params.getRight().get(), ctx).orElse(""));
+        }
+        return args;
+    }
+
+    /**
+     * Get the parameter type and name pair given the paramter symbol.
+     *
+     * @param param parameter symbol.
+     * @param ctx   Lang Server Operation context
+     * @return {@link Optional<String>} Type and name pair.
+     */
+    public static Optional<String> getParamTypeAndNamePair(ParameterSymbol param, BallerinaCompletionContext ctx) {
```

Comment:
Does the name of this method still valid?

Reply:
This need to be fixed